bin/makemessages.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -eu
+
+# ensure script runs from correct location
+
+cd /home/calitp/app
```

Comment:
This isn't a pattern in our other helper scripts. I think we can assume (and we document as such) that they are run like `bin/script.sh`